sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -85,8 +95,42 @@ def _evaluate_by_index(self, y_true, y_pred, multioutput, **kwargs):
             )
         return out_series
 
+    def _check_alpha(self, alpha):
```

Comment:
Not sure this is needed anymore. Alphas are now retrieved directly from the y_pred dataframe rather than being user inputs, so there should never be an invalid alpha.

Reply:
I guess - and we also do input checks on `y_pred`, which I think also checks the alpha?